pkg/kudoctl/cmd/plan/plan_status.go

```diff
@@ -60,12 +60,17 @@ func status(kc *kudo.Client, options *Options, ns string) error {
 		return ""
 	}
 
-	rootDisplay := fmt.Sprintf("%s (Operator-Version: \"%s\" Active-Plan: \"%s\")", instance.Name, instance.Spec.OperatorVersion.Name, lastPlanStatus.Name)
+	rootDisplay := fmt.Sprintf("%s (Operator-Version: \"%s\" Active-Plan: \"%s\", last updated: \"%s\")", instance.Name, instance.Spec.OperatorVersion.Name, lastPlanStatus.Name, instance.Status.AggregatedStatus.LastUpdated.Format("2006-01-02 15:04:05"))
```

Comment:
Looking at this, this would fail if the new KUDO version tries to access an existing instance that doesn't have the LastUpdated field set yet. Should be fine for clean installations of the new KUDO versions, but as it is a pointer and can be nil, it should be checked.

Reply:
Good point, I should have thought of this! 🙈